### DISCUSSION TITLE
Swin UNETR with 2D image

### DISCUSSION THREAD

**Discussion by YHYeooooong** [Project-MONAI/MONAI#5608] [Q&A] [answered]
Thank you for sharing great work!

I wonder if the Swin UNETR can training and inferencing with 2D image shapes something like (height, width, channel)?

I knew the proposed Swin UNETR trained with 3D images in the paper, but in the model code, it seems to can work on 2D images.
https://github.com/Project-MONAI/MONAI/blob/acffc2c7b1b9aaf29e1ee0710e9b17acee0dcac2/monai/networks/nets/swin_unetr.py#L588

Thanks!

**Comment by tangy5** [comment, marked as answer]
HI @YHYeooooong , thanks for the question. 
Yes, the implementation of Swin UNETR support 2D images as input, just like UNet or other segmentation networks. 
But as you mentioned, we do not have a demo or preliminary benchmark for 2D images. You are very welcome to use for 2D images and see how it work. 

Thanks.

**Reply by YHYeooooong** [reply]
Thanks for fast reply!
I will try ☺
